src/Spout/Writer/ODS/Manager/WorksheetManager.php

```diff
@@ -209,6 +209,11 @@ private function getCellXML(Cell $cell, $styleIndex, $numTimesValueRepeated)
             $data .= ' office:value-type="string" calcext:value-type="error" office:value="">';
             $data .= '<text:p>' . $cell->getValueEvenIfError() . '</text:p>';
             $data .= '</table:table-cell>';
+        } elseif ($cell->isDate()) {
+            $dateStr = $cell->getValue()->format('Y-m-d');
```

Comment:
Spout should not decide the date format (`Y-m-d` is a very opinionated choice). I believe that this should be let to the user to convert the datetime to a formatted date (or more complex and unsupported at the moment: allow the user to specify the date format as part of the cell style)